src/main/java/sirius/biz/analytics/scheduler/BaseEntityBatchEmitter.java

```diff
@@ -91,11 +92,11 @@ public <E extends B> void computeBatches(Class<E> type,
                 return;
             }
 
-            JSONObject batch = new JSONObject();
+            ObjectNode batch = Json.createObject();
             batch.put(TYPE, Mixing.getNameForType(type));
             batch.put(EXPECTED_COUNT, counter.get());
-            batch.put(START_ID, lastLimit.get());
-            batch.put(END_ID, nextLimit.get());
+            batch.putPOJO(START_ID, lastLimit.get());
+            batch.putPOJO(END_ID, nextLimit.get());
```

Comment:
Why do we need `putPOJO` here?

Reply:
Because `lastLimit` and `nextLimit` iss of type `I` here and Java cant decide whether or not `I` will match any of the explicit types supported by the `put`-methods.